services/background-worker/test/Test/Wire/BackendNotificationPusherSpec.hs

```diff
@@ -331,6 +331,7 @@ spec = do
           backendNotificationsConfig = BackendNotificationsConfig 1000 500000 1000
 
       backendNotificationMetrics <- mkBackendNotificationMetrics
+      workerRunningGauge <- mkWorkerRunningGauge
```

Comment:
Some assertion / test to check the current state of the gauge would be nice.

Reply:
This is tough to check because making this metric go to 0 means causing RabbitMQ outage for long enough that we can detect this change in metrics. I don't think we can do that easily.

Reply:
Okay, testing the metrics is probably not crucial.